wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceInfoViewModel.swift

```diff
@@ -228,3 +194,37 @@ extension DeviceInfoViewModel {
         )
     }
 }
+
+extension E2EIdentityCertificateStatus {
+    func titleForStatus() -> String {
+        switch self {
+        case .notActivated:
+            return L10n.Localizable.Device.Details.Section.E2ei.Status.notActivated
```

Comment:
You can create a `typealias` with `L10n.Localizable.Device.Details.Section.E2ei.Status`. This might improve readability a bit and reduce duplicate code.